subsys/bluetooth/controller/ll_sw/nordic/lll/lll_peripheral_iso.c

```diff
@@ -647,13 +684,10 @@ static void isr_rx(void *param)
 		has_tx = 1U;
 
 		/* Adjust nesn when flushing Tx */
-		/* FIXME: When Flush Timeout is implemented */
 		if (cis_lll->tx.bn_curr <= cis_lll->tx.bn) {
 			lll_flush_tx(cis_lll);
 		}
-
-		/* Start transmitting new burst */
-		cis_lll->tx.bn_curr = 1U;
+		cis_lll->tx.bn_curr = 1; /* FIXME shouldn't this be done in the prepare_cb? */
```

Comment:
revert to LHS (i see no change done than remove existing comment).

You could add additional comment that, as a peripheral bn_curr is reset after Radio Rx (instead of as in central where it transmits first in an event)